core/controllers/learner_dashboard_test.py

```diff
@@ -246,7 +246,6 @@ class LearnerDashboardFeedbackThreadHandlerTests(test_utils.GenericTestBase):
     def setUp(self):
         super(LearnerDashboardFeedbackThreadHandlerTests, self).setUp()
         self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
-
```

Comment:
Keep this newline.

Reply:
OK, I'll add that new line in! Also, I think it would be best if we could merge this PR. I am working with someone else on finishing another unit test, so that other one will be the one we bring to 100%. This test was mainly to go through the full process of PR, as recommended by Sean Lip.

Reply:
Done